js/package.json

```diff
@@ -14,12 +14,15 @@
     }
   ],
   "dependencies": {
+    "accept-language-parser": "^1.5.0",
     "bluebird": "~3.5.1",
     "body-parser": "~1.18.3",
+    "cookie-parser": "^1.4.3",
+    "eureka-js-client": "~4.4.1",
     "express": "~4.16.3",
     "express-session": "~1.15.6",
     "express-ws": "~4.0.0",
-    "eureka-js-client": "~4.4.1",
+    "glob": "^7.1.3",
```

Comment:
Please make these ~ versions instead of ^ versions, for ease of maintenance over time.

Reply:
Done